src/mcmc/gibbs.jl

```diff
@@ -80,11 +130,11 @@ function get_conditioned_gibbs(context::GibbsContext, vns::AbstractArray{<:VarNa
 end
 
 function is_target_varname(context::GibbsContext, vn::VarName)
-    return Iterators.any(
-        Iterators.map(target -> subsumes(target, vn), context.target_varnames)
-    )
+    return is_target_varname(context, DynamicPPL.getsym(vn))
 end
 
+is_target_varname(::GibbsContext{T}, vn_symbol::Symbol) where {T} = vn_symbol in T
```

Comment:
I'd highly recommend just making this one function that does
```julia
is_target_varname(context::GibbsContext{T}, vn::VarName{sym}) where {sym} = sym in T
```
Otherwise we're heavily relying on Julia's constant propagation hitting => less likely to get inlined as program size grows

Reply:
Agreed, done.